examples/crd-client/main.go

```diff
@@ -30,19 +31,30 @@ import (
 )
 
 const (
-	gameServerImage = "GAMESERVER_IMAGE"
+	gameServerImage      = "GAMESERVER_IMAGE"
+	isHelmTest           = "IS_HELM_TEST"
+	gameserversNamespace = "GAMESERVERS_NAMESPACE"
 
 	defaultImage = "gcr.io/agones-images/udp-server:0.19"
+	defaultNs    = "default"
 )
 
 func main() {
 	viper.AllowEmptyEnv(true)
 	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
 
-	pflag.String(gameServerImage, viper.GetString(gameServerImage), "The Address to bind the server grpcPort to. Defaults to 'localhost'")
+	pflag.String(gameServerImage, defaultImage, "The Address to bind the server grpcPort to. Defaults to 'localhost'")
 	viper.SetDefault(gameServerImage, defaultImage)
 	runtime.Must(viper.BindEnv(gameServerImage))
-	viper.GetString(gameServerImage)
+
+	pflag.Bool(isHelmTest, false, "Is helm test - shutdown GameServer at the end of test. Defaults to false")
```

Comment:
Can we rephrase - "Defines whether GameServer should be shut down at the end of the test or not"?

Reply:
added your definition.